jablib/src/main/java/org/jabref/logic/importer/util/FileFieldParser.java

```diff
@@ -56,7 +56,7 @@ public List<LinkedFile> parse() {
             return files;
         }
 
-        if (LinkedFile.isOnlineLink(value.trim())) {
+        if (URLUtil.isURL(value.trim())) {
```

Comment:
Why this change? The code of `org.jabref.model.entry.LinkedFile#isOnlineLink(java.lang.String)` is (intentionally?) different from `URLUtil.isURL`

If there is a reason: Add a comment explaining that `...isOnlineLink(java.lang.String)` is not chosen but URLUtil...